tasks.py

```diff
@@ -0,0 +1,9 @@
+from invoke import task
```

Comment:
I haven't used `invoke` before. What does this do that calling mypy in as a shell command inside GitHub actions does not?

Reply:
Invoke's tasks.py acts like a makefile, allowing you to build a collection of standardised commands you can call both in or outside of CI.

Reply:
It is not strictly essential here, but I find it useful.

Reply:
Would you like me to remove it?

Reply:
Let's give it a try. Seems like a light addition for now. Easy to remove down the line if it becomes a bother.